integrations/Lytics/browser.js

```diff
@@ -0,0 +1,141 @@
+// disabled these for Lytics js tag
+/* eslint-disable no-plusplus */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-sequences */
+/* eslint-disable yoda */
+/* eslint-disable prefer-spread */
+// disabling these eslint which are caused by the Lytics js tag
+
+/* eslint-disable prefer-rest-params */
+/* eslint-disable no-var */
+/* eslint-disable vars-on-top */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-shadow */
+/* eslint-disable no-void */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable one-var */
+/* eslint-disable lines-around-directive */
+/* eslint-disable strict */
+import logger from "../../utils/logUtil";
+
+class Lytics {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.stream = config.stream;
+    this.blockload = config.blockload;
+    this.loadid = config.loadid;
+    this.name = "LYTICS";
+  }
+
+  loadLyticsScript() {
+    !(function () {
```

Comment:
We can keep either **!** or **(func call)**
https://stackoverflow.com/questions/3755606/what-does-the-exclamation-mark-do-before-the-function

Reply:
Done